Sources/ArcGISToolkit/Components/Popups/PopupView.swift

```diff
@@ -35,179 +35,126 @@ import ArcGIS
-        self.isPresented = isPresented
-    }
-    
-    /// The `Popup` to display.
-    private let popup: Popup
-    
-    /// A Boolean value specifying whether a "close" button should be shown or not. If the "close"
-    /// button is shown, you should pass in the `isPresented` argument to the initializer,
-    /// so that the the "close" button can close the view.
-    var showCloseButton = false
+    /// A binding to the popup currently being presented in the navigation layer.
+    @Binding private var presentedPopup: Popup?
     
-    /// A Boolean value indicating whether the deprecated `PopupView.showCloseButton(_:)`
-    /// modifier is applied.
+    /// The popups that have been presented in the navigation layer.
     ///
-    /// Once the modifier is removed this property can be removed and the presence or lack of a value for
-    /// `isPresented` should be the sole criteria to show or hide the close button.
-    var showCloseButtonDeprecatedModifierIsApplied = false
+    /// There is one popup for every `InternalPopupView` in the current
+    /// navigation stack.
+    @State private var presentedPopups: [Popup] = []
     
-    /// The visibility of the popup header.
-    var headerVisibility: Visibility = .automatic
+    /// The model for the navigation layer.
+    @State private var navigationLayerModel: NavigationLayerModel?
     
-    /// The result of evaluating the popup expressions.
-    @State private var evaluation: Evaluation?
+    /// The visibility of the close button.
+    var closeButtonVisibility: Visibility = .automatic
     
     /// A binding to a Boolean value that determines whether the view is presented.
-    private var isPresented: Binding<Bool>?
+    ///
+    /// This property can be removed once the deprecated
+    /// `init(popup:isPresented:)` initializer is removed.
+    var isPresented: Binding<Bool>?
+    
+    /// Creates a `PopupView` to display a given popup.
+    /// - Parameter popup: A binding to the popup that view displays. When `popup` is non-`nil`, the `PopupView` is displayed.
+    /// - Since: 200.8
+    public init(popup: Binding<Popup?>) {
+        self._presentedPopup = popup
+    }
     
     public var body: some View {
-        VStack(alignment: .leading) {
-            if headerVisibility != .hidden {
-                HStack {
-                    if !popup.title.isEmpty {
-                        Text(popup.title)
-                            .font(.title)
-                            .fontWeight(.bold)
-                    }
-                    Spacer()
-                    if (showCloseButtonDeprecatedModifierIsApplied && showCloseButton)
-                        || (!showCloseButtonDeprecatedModifierIsApplied && isPresented != nil) {
-                        XButton(.dismiss) {
-                            isPresented?.wrappedValue = false
+        VStack(spacing: 0) {
+            if let popup = presentedPopups.first {
+                NavigationLayer { model in
```

Comment:
```suggestion
                EmbeddedNavigationStack { model in
```
The NavigationLayer name doesn't need to change in this PR, but I do think we should come up with a different name. This is just a possible suggestion. "Embedded" in the sense that it works within an existing system level navigation stack.

Reply:
Added this suggestion to Apollo 1235.